packages/mui-material/src/Autocomplete/Autocomplete.test.js

```diff
@@ -2622,5 +2622,27 @@ describe('<Autocomplete />', () => {
       fireEvent.keyDown(textbox, { key: 'Backspace' });
       expect(container.querySelectorAll(`.${chipClasses.root}`)).to.have.length(2);
     });
+
+    it('should not crash if renderTags is null', () => {
```

Comment:
Something is off with the test. It passes on master, so it likely means it is not testing the original issue.

Reply:
@mnajdova  Hey I didn't quite understand that. Could you please help explain another way?

Reply:
The test is passing on master (which shouldn't be if we are really testing the original issue). So I would assume something needs to be updated.

Reply:
Oh okay, I'll see what's the problem. Thank you.

Reply:
@mnajdova In the latest version of the library, we don't have this problem, as we can observe here https://codesandbox.io/s/p7njj6?file=/demo.tsx (It's the same example just with latest library)
But this issue is using version 4 of the library. 
So, maybe we should close the issue?